tests/unit_tests/data_model/test_metadata_collector.py

```diff
@@ -472,3 +472,33 @@ def test_clean_meta_data():
 
     collector = metadata_collector.MetadataCollector({})
     assert collector.clean_meta_data(pre_clean) == post_clean
+
+
+def test_fill_contact_meta_from_system(args_dict_site, caplog):
+    """Test filling contact metadata from system when no name is provided"""
+    contact_dict = {}
+    collector = metadata_collector.MetadataCollector(args_dict=args_dict_site)
+
+    with caplog.at_level(logging.WARNING):
+        collector._fill_contact_meta(contact_dict)
+
+    assert "No user name provided, take user info from system level." in caplog.text
+    assert contact_dict["name"] == getpass.getuser()
```

Comment:
Will you ever run the tests on the AIV system (or something similar)? This line would fail in that case, so perhaps you want to add a try..except here as well.

Reply:
At this point, it is not planned to run unit tests.

Anything, this is a good remark and I protected the statement with a try .. except: pass statement.